### PR TITLE
Migrate to dotnet foundation

### PR DESCRIPTION
**Pull request by marabooy**
Adds license and Readme updates for migration to .Net foundation

### CODE REVIEW

File: src/Microsoft.OData.Extensions.Client.Abstractions/Microsoft.OData.Extensions.Client.Abstractions.Nightly.nuspec [hunk 1]
```diff
@@ -4,8 +4,8 @@
         <id>Microsoft.OData.Extensions.Client.Abstractions</id>
         <title>Microsoft Extensions OData Client.Abstractions</title>
         <version>$VersionFullSemantic$-Nightly$NightlyBuildVersion$</version>
-        <authors>Microsoft</authors>
-        <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
+        <authors>.NET Foundation</authors>
+        <copyright>&#169; .NET Foundation and Contributors.</copyright>
```
Comment (gathogojr) [inline src/Microsoft.OData.Extensions.Client.Abstractions/Microsoft.OData.Extensions.Client.Abstractions.Nightly.nuspec:8]: @marabooy You retained the "All rights reserved." part in these other 2 .nuspec files:
- src/Microsoft.OData.Extensions.Migration/Microsoft.OData.Extensions.Migration.Nightly.nuspec
- src/Microsoft.OData.Extensions.Migration/Microsoft.OData.Extensions.Migration.Release.nuspec

File: src/Microsoft.OData.Extensions.Client.Abstractions/Microsoft.OData.Extensions.Client.Abstractions.Release.nuspec [hunk 2]
```diff
@@ -4,8 +4,8 @@
         <id>Microsoft.OData.Extensions.Client.Abstractions</id>
         <title>Microsoft Extensions OData Client.Abstractions</title>
         <version>$VersionNuGetSemantic$</version>
-        <authors>Microsoft</authors>
-        <copyright>&#169; Microsoft Corporation. All rights reserved.</copyright>
+        <authors>.NET Foundation</authors>
+        <copyright>&#169; .NET Foundation and Contributors.</copyright>
```
Comment (gathogojr) [inline src/Microsoft.OData.Extensions.Client.Abstractions/Microsoft.OData.Extensions.Client.Abstractions.Release.nuspec:8]: @marabooy You retained the "All rights reserved." part in these other 2 .nuspec files:
- src/Microsoft.OData.Extensions.Migration/Microsoft.OData.Extensions.Migration.Nightly.nuspec
- src/Microsoft.OData.Extensions.Migration/Microsoft.OData.Extensions.Migration.Release.nuspec

File: src/AssemblyInfo/AssemblyInfoCommon.cs [hunk 3]
```diff
@@ -12,13 +12,13 @@
...
-[assembly: AssemblyCompany("Microsoft Corporation")]
+[assembly: AssemblyCompany(".NET Foundation")]
 // If you want to control this metadata globally but not with the VersionProductName property, hard-code the value below.
 // If you want to control this metadata at the individual project level with AssemblyInfo.cs, comment-out the line below.
 // If you leave the line below unchanged, make sure to set the property in the root build.props, e.g.: <VersionProductName Condition="'$(VersionProductName)'==''">Your Product Name</VersionProductName>
 // [assembly: AssemblyProduct("%VersionProductName%")]
-[assembly: AssemblyCopyright("Copyright (c) Microsoft Corporation. All rights reserved.")]
-[assembly: AssemblyTrademark("Microsoft and Windows are either registered trademarks or trademarks of Microsoft Corporation in the U.S. and/or other countries.")]
+[assembly: AssemblyCopyright("Copyright (c) .NET Foundation and Contributors. All rights reserved.")]
+[assembly: AssemblyTrademark(".NET Foundation and Windows are either registered trademarks or trademarks of Microsoft Corporation in the U.S. and/or other countries.")]
```
Comment (gathogojr) [inline src/AssemblyInfo/AssemblyInfoCommon.cs:21]: @marabooy Is this statement correct/factual?

Comment (marabooy) [reply src/AssemblyInfo/AssemblyInfoCommon.cs:21]: For this had used a simple find and replace. currently fixing this.

**Comment by marabooy** [comment]
> Are you migrating the build pipeline to dotnet foundation as well in this PR? If not then probably you need to change the tittle of the PR.

Yes the pipeline was added in this monstrosity of  pr. once merged will perform the last step to add the badges.

**Comment by marabooy** [comment]
@paulodero Until it is merged didn't want to adversely affect others who rely on the current pipeline 
A manual run shows that this builds and generates the artifacts 
image

image

File: README.md [hunk 4]
```diff
@@ -21,6 +21,14 @@ For more information about OData, please refer to the following resources:
 - [Build an OData v4 Service with OData WebApi Library](http://odata.github.io/WebApi/#01-02-getting-started)
```
Comment (KenitoInc) [inline README.md:21]: Both of these links lead to the odata "home page". Update the correct links

File: README.md [hunk 5]
```diff
@@ -111,12 +110,21 @@ There are many ways for you to contribute to OData .NET. The easiest way is to p
 - Feedback: Please send mails to [odatafeedback@microsoft.com](mailto:odatafeedback@microsoft.com).
 - Team blog: Please visit [http://blogs.msdn.com/b/odatateam/](http://blogs.msdn.com/b/odatateam/) and [http://www.odata.org/blog/](http://www.odata.org/blog/).
```
Comment (KenitoInc) [inline README.md:111]: This link doesn't work http://blogs.msdn.com/b/odatateam

File: README.md [hunk 6]
```diff
@@ -111,12 +110,21 @@ There are many ways for you to contribute to OData .NET. The easiest way is to p
...
 
+### High Level roadmap for OData Connected Service
```
Comment (KenitoInc) [inline README.md:113]: Did you mean Odata Extensions?

File: README.md [hunk 7]
```diff
@@ -111,12 +110,21 @@ There are many ways for you to contribute to OData .NET. The easiest way is to p
...
+
 ### Thank you
 
 We’re using NDepend to analyze and increase code quality.
 
+### Other related Projects
+
+*  [Microsoft OData Libraries](https://github.com/OData/odata.net)
+*  [Microsoft WebAPi Library](https://github.com/OData/WebApi)
+*  [Microsoft Restier](https://github.com/OData/RESTier)
```
Comment (KenitoInc) [inline README.md:124]: We could add OCS here

File: README.md (hunk 6 above)
Comment (marabooy) [reply README.md:113]: 👍

**Pull request merged**